src/config.rs

```diff
@@ -180,13 +184,27 @@ impl BuilderConfig {
         }
     }
 
-    /// Connect to the provider using the configuration.
-    pub async fn connect_provider(&self) -> Result<Provider, ConfigError> {
+    /// Connect to host rpc provider.
+    pub async fn connect_host_provider(&self) -> Result<Provider, ConfigError> {
         let builder_signer = self.connect_builder_signer().await?;
+        BuilderConfig::connect_provider(builder_signer, self.host_rpc_url.clone()).await
+    }
+
+    /// Connect to rollup rpc provider.
+    pub async fn connect_ru_provider(&self) -> Result<Provider, ConfigError> {
```

Comment:
this is probably outside of scope, but given the builder never needs to submit rollup txns, it technically doesn't need a signer

Reply:
🙂 https://github.com/init4tech/builder/pull/31/commits/daeb424e03d11440fe0c7b3985554b61fcc2da04